beebop/filestore.py

```diff
@@ -198,51 +184,27 @@ def microreact_json(self, p_hash, cluster) -> str:
         """
         return str(
             PurePath(
-                self.output(p_hash),
-                f"microreact_{cluster}",
-                (f"microreact_{cluster}.microreact"),
-            )
-        )
-
-    def network_output_csv(self, p_hash) -> str:
-        """
-        :param p_hash: [project hash]
-        :return str: [path to network csv file]
-        """
-        return str(
-            PurePath(self.output(p_hash), "network", "network_cytoscape.csv")
-        )
-
-    def network_output_component(self, p_hash, component_number) -> str:
-        """
-        :param p_hash: [project hash]
-        :param component_number: [component number,
-            which is the same as cluster number]
-        :return str: [path to network component file]
-        """
-        return str(
-            PurePath(
-                self.output(p_hash),
-                "network",
-                f"network_component_{component_number}.graphml",
+                self.output_visualisations(p_hash, cluster),
+                f"visualise_{cluster}.microreact",
             )
         )
 
-    def pruned_network_output_component(self, p_hash, component_number) -> str:
+    def pruned_network_output_component(
+        self, p_hash, component: str, cluster
+    ) -> str:
         """
         [Generates the path to the pruned network component file
         for the given project hash and component number.]
 
         :param p_hash: [project hash]
-        :param component_number: [component number,
-            which is the same as cluster number]
+        :param component: [component number,
+            which is the same as raw cluster number]
+        :param cluster: [assigned cluster number]
         :return str: [path to pruned network component file]
         """
-        return str(
-            PurePath(
-                self.output(p_hash),
-                "network",
-                f"pruned_network_component_{component_number}.graphml",
+        return str(PurePath(
+            self.output_visualisations(p_hash, cluster),
+            f"pruned_visualise_{cluster}_component_{component}.graphml",
```

Comment:
Why is the cluster now included here as well as the component?

Reply:
as per discussion